### PR TITLE
Fix the behaviour of Ctrl+E and Ctrl+W when switching preview by RIGHTCLICK

### PR DESCRIPTION
**Pull request by asmsuechan**
## before
Nothing happens when `Ctrl+E` is pressed on RIGHTCLICK mode. Also `Ctrl+W` doesn't work.

## after
They work well.

### CODE REVIEW

File: browser/main/Detail/index.js [hunk 1]
```diff
@@ -14,7 +14,12 @@ class Detail extends React.Component {
     super(props)
 
     this.focusHandler = () => {
+      if (this.refs.root.state.status === 'CODE') return
       this.refs.root != null && this.refs.root.focus()
```
Comment (sota1235) [inline browser/main/Detail/index.js]: [NOT MUST] It seems to be not necessary. Can you remove this line?

Comment (sota1235) [inline browser/main/Detail/index.js]: Sorry, we can't remove this line.
Maybe fixing like below is better.

```javascript
if (this.refs.root != null) {
  this.refs.root.focus()
}
```

File: browser/main/Detail/index.js [hunk 2]
```diff
@@ -14,7 +14,12 @@ class Detail extends React.Component {
...
+      let { config } = this.props
+      if (config.editor.switchPreview === 'RIGHTCLICK') {
+        this.refs.root.setState({status: ''})
```
Comment (sota1235) [inline browser/main/Detail/index.js]: [NOT MUST] It is not good way to use React component.
Can you keep `status` state on this component and pass it as props to `this.refs.root`?

**Comment by asmsuechan** [comment]
close this once.

**Pull request closed without merging**